### PR TITLE
cilium-cni: Scripts to install and uninstall CNI configuration

### PR DESCRIPTION
**Pull request by tgraf**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -56,9 +56,10 @@ apt-get purge --auto-remove -y gcc make bison flex git curl xz-utils && \
 # Needed for system minimal requirements checkers
 apt-get install -y --no-install-recommends libgcc-5-dev binutils && \
 apt-get clean && \
-rm -fr /root /var/lib/apt/lists/* /tmp/* /var/tmp/* /usr/local/go && \
-echo '#!/usr/bin/env bash\ncp /opt/cni/bin/cilium-cni /tmp/cni/bin && /usr/bin/cilium-agent $@' > /home/with-cni.sh && \
-chmod +x /home/with-cni.sh
+rm -fr /root /var/lib/apt/lists/* /tmp/* /var/tmp/* /usr/local/go
+
+ADD plugins/cilium-cni/cni-install.sh /cni-install.sh
```
Comment (aanm) [inline Dockerfile:61]: You can copy the file from ` /tmp/cilium-net-build/src/github.com/cilium/cilium/plugins/cilium-cni/cni-install.sh` before the `rm -fr` command.

**Review by aanm — DISMISSED** [review]

File: Dockerfile (hunk 1 above)
Comment (tgraf) [reply Dockerfile:61]: What do you mean? What file? What is wrong with `ADD`?

Comment (aanm) [reply Dockerfile:61]: We can optimize the image size later.

**Review by aanm — APPROVED** [review]

**Review by aalemayhu — COMMENTED** [review]
Looks good.

**Pull request merged**